.circleci/config.yml

```diff
@@ -20,3 +37,73 @@ workflows:
             ignore:
             - main
             - master
+
+    - architect/push-to-app-collection:
+        context: architect
+        name: capa-app-collection
+        app_name: logging-operator
```

Comment:
Name is wrong here, and should probably be something like `observability-platform-api`

Reply:
True

Reply:
```suggestion
        app_name: observability-platform-api
```